src/main/cpp/t01_min.cpp

```diff
@@ -24,8 +24,16 @@ using namespace std;
 //function min 
 
 //function min4 
-
+int min(int h1, int h2) {
+	if (h1 < h2) return h1;
+	else return h2;
+}
+int min4(int z1, int z2, int z3, int z4) {
+	int min4 = min(min(z1, z2), min(z3, z4));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'min4' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594246)